lib/cluster.js

```diff
@@ -27,7 +27,19 @@ function Worker(options) {
   if (options === null || typeof options !== 'object')
     options = {};
 
-  this.suicide = undefined;
+  this.exitedVoluntarily = undefined;
+
+  Object.defineProperty(this, 'suicide', {
+    get: function() {
+      // TODO: Print deprecation message.
+      return this.exitedVoluntarily;
+    },
+    set: function(val) {
+      // TODO: Print deprecation message.
+      this.exitedVoluntarily = val;
+    }
+  });
```

Comment:
I think this got lost in a commit, so:

> Note: it's possible this could break if people are trying to enumerate the values of a worker object. I remember this happening with something http, though I think it will be less likely here.